WalletWasabi/JsonConverters/Uint256JsonConverter.cs

```diff
@@ -24,7 +24,7 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 		/// <inheritdoc />
 		public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
 		{
-			writer.WriteValue(((uint256)value).ToString());
+			writer.WriteValue(((uint256)value)?.ToString());
```

Comment:
I have no context to know how correct this is. It looks correct if we want to write `null` when the value comes `null`.